app/main.py

```diff
@@ -13,6 +13,8 @@ def feed(self):
             self.is_hungry = False
             food_points, self.appetite = self.appetite, 0
```

Comment:
we can't change  - "appetite", this value must remain unchanged.
You need to return the number of units eaten depending on the status of "is_hungry"